microsoft-edge/accessibility/build/index.md

```diff
@@ -135,7 +135,7 @@ These are blogs about accessibility.
 
 ### Simply Accessible
 
-[Simply Accessible](http://simplyaccessible.com/articles) is a team of accessibility specialists providing accessibility training, consulting and more to change the perception of accessibility on the web.  Their [Articles](http://simplyaccessible.com/articles) section discusses best practices for web accessibility, accessible design, and more.
+[Simply Accessible](http://simplyaccessible.com/articles) is a team of accessibility specialists providing accessibility training, consulting and more to change the perception of accessibility on the web.  Their [Articles](http://simplyaccessible.com) section discusses best practices for web accessibility, accessible design, and more.
```

Comment:
I'm not sure which articles we were referring to here, but I don't know if they're still available. It might be worth checking the new domain a bit, trying to find them. If we can't find them, then let's remove the section.

Reply:
Removed the section

Reply:
fixed in new pr